types/operator.go

```diff
@@ -5,13 +5,19 @@ type OperatorID = uint64
 
 // Operator represents an SSV operator node
 type Operator struct {
-	OperatorID OperatorID
-	PubKey     []byte `ssz-size:"48"`
+	OperatorID    OperatorID
+	SharePubKey   []byte `ssz-size:"48"`
+	NetworkPubKey []byte `ssz-size:"294"`
 }
 
-// GetPublicKey returns the public key with which the node is identified with
-func (n *Operator) GetPublicKey() []byte {
-	return n.PubKey
+// GetSharePublicKey returns the share public key
+func (n *Operator) GetSharePublicKey() []byte {
+	return n.SharePubKey
+}
```

Comment:
> But I think this should be done in another PR.
Aite, too many diffs are bad at this point 👍 

> Even though spec may not care about memory, since the implementation is forced to be aligned to spec, I think we should care and implement it in the future PR.
Not important for me, but we can. If @moshe-blox will feel this is important then we will do it for sure.